server/src/test/Parsing.test.ts

```diff
@@ -745,5 +742,30 @@ Music/wedrujacy_wiatr/tam_gdzie_miesiac_oplakuje_swit/flac/02_-_tam_gdzie_miesia
             );
             done();
         });
+        it('should parse local_variable_declarations with multiple potential types without error', (done) => {
+            checkInputValidityForRule(
+                (parser) => parser.local_variable_decl(),
+                `Type1 || Type2 || Collection myVariable`,
+            );
+            done();
+        });
+        it('should parse enums with attributes without error', (done) => {
+            checkInputValidityForRule(
+                (parser) => parser.yenum(),
+                `enum MyEnum
+                attributes {
+                    Number myAttr
+                    --> documentation """"myAttr's documentation"""
+                } {
+                    ENUM_VALUE_1
+                    --> myAttr 12,
+
+                    ENUM_VALUE_2
+                    --> myAttr 13
+                }
+                --> documentation """"This is MyEnum's documentation""";`,
```

Comment:
_Quadruple_ `"`-s in two places?

Reply:
Should be three `"` instead of four, yeah. I change that.

Reply:
Isn’t it strange that the test did not care more about that? xD

Reply:
oh maybe the fourth `"` was just embedded into the text

Reply:
Two options here:
- it is valid YML
- the grammar needs improvements

Reply:
no I think it’s like a `"\"foo"`.

Reply:
It seems it is not valid YML.
I'll improve the grammar in another PR.